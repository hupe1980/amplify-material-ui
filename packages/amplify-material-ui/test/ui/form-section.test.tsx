import * as React from 'react';
import { render } from '@testing-library/react';
import { FormSection } from '../../src';
import { withTheme } from '../helpers/withTheme';

describe('form-section', () => {
  it('should be rendered correctly', () => {
    const { asFragment } = render(
      withTheme(
        <FormSection>
          <div />
        </FormSection>,
      )
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
