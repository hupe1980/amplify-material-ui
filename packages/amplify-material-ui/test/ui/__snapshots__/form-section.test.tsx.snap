// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form-section should be rendered correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthXs css-nzinn5-MuiContainer-root"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 makeStyles-paper-1 css-1ps6pg7-MuiPaper-root"
    >
      <div />
    </div>
  </div>
</DocumentFragment>
`;
